.gitignore

```diff
@@ -9,3 +9,5 @@ gh-md-toc
 web/public/
 /drand
 cover*
+.idea
+**/*.pb.go
```

Comment:
We would prefer not to keep them on the repo, as it can lead to errors between the definition of the messages (.proto files) and the auto generated code. Building process is quite simple and fast, and it is now wired up inside Makefile commands.

Reply:
We want to make sure that the versions built in CI and in deployed docker images are from known versions of the generated code and are reproducible.

this is the expected convention for this sort of golang code: https://github.com/golang/go/issues/31239#issuecomment-480020869